integration-tests/src/test/java/oracle/kubernetes/operator/ItSitConfigDomainInImage.java

```diff
@@ -0,0 +1,277 @@
+// Copyright (c) 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.logging.Level;
+
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.LoggerHelper;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+
+/**
+ * JUnit test class used for testing configuration override use cases for Domain In Image.
+ */
+public class ItSitConfigDomainInImage extends SitConfig {
+
+  private static String testClassName;
+  private static int testNumber;
+  private static Operator operator1;
+  private static Domain domain;
+  private static String sitconfigTmpDir = "";
+  private static String mysqltmpDir = "";
+  private static String configOverrideDir = "";
+  private static String mysqlYamlFile = "";
+  private static String domainNS;
+  private static String testprefix = "sitconfigdomaininimage";
+  private static String mysqldbport;
+  private static String adminpodname;
+  private static String JDBC_URL;
+  private static StringBuffer namespaceList;
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception when the initialization, creating directories , copying files and domain
+   *                   creation fails.
+   */
+
+  @BeforeClass
+  public static void staticPrepare() throws Exception {
+    if (FULLTEST) {
+      testClassName = new Object() {
+      }.getClass().getEnclosingClass().getSimpleName();
+      initialize(APP_PROPS_FILE, testClassName);
+      TEST_RES_DIR = getProjectRoot() + "/integration-tests/src/test/resources/";
+      testNumber = getNewSuffixCount();
+    }
+  }
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception when the initialization, creating directories , copying files and domain
+   *                   creation fails.
+   */
+  @Before
+  public void prepare() throws Exception {
```

Comment:
The code seems to be same for both the classes, can u move this to super class by adding a new method with args and call that method here